assets/js/agent/app.vue

```diff
@@ -56,27 +56,85 @@
     created: function() {
       this.loadUser();
       this.loadChannels();
+      this.syncSubscriptions();
     },
     methods: {
       loadChannels: function() {
         var $data = this.$data;
         var me = this;
 
         this.realmq.channels.list().then(function (channelList) {
-          channelList.items.sort(function (a, b) {
+          $data.channels = channelList.items.sort(function (a, b) {
             return new Date(a.createdAt) < new Date(b.createdAt) ? 1 : -1;
```

Comment:
Not part of this PR but its not necessary to cast to Date as iso date time strings are lexical sortable.